src/components/Call/Call.js

```diff
@@ -53,6 +53,13 @@ export default function Call() {
                 : 'self-view alone'
             }>
             <video autoPlay muted playsInline ref={localVideoElement} />
+            <div className="username">
+              You (
+              {localParticipant?.user_name
+                ? localParticipant?.user_name
+                : localParticipant?.user_id}
```

Comment:
`localParticipant?.user_name || localParticipant?.user_id` is a little easier to read imo

Reply:
I neeeeeeed JavaScript operators to be easier to google, I was trying `&&` and it didn't work and I was like FINE I'll do it the hard way. Thank you for this 👯